src/test/java/pl/szczesniak/dominik/whattowatch/movies/domain/model/MoviesToWatchServiceTest.java

```diff
@@ -0,0 +1,122 @@
+package pl.szczesniak.dominik.whattowatch.movies.domain.model;
+
+import org.junit.jupiter.api.Test;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.UserService;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.UserServiceConfiguration;
+import pl.szczesniak.dominik.whattowatch.users.infrastructure.persistence.InMemoryUserRepository;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static pl.szczesniak.dominik.whattowatch.movies.domain.model.TestMoviesToWatchServiceConfiguration.moviesToWatchService;
+
+class MoviesToWatchServiceTest {
+
+//    MoviesToWatchService mocked = mock();
+    private final UserService userService = new UserServiceConfiguration().userService(new InMemoryUserRepository());
+    private final MoviesToWatchService tut = moviesToWatchService(userService);
+
+    @Test
+    void user_should_add_movie_to_his_list() {
+        // given
+        userService.createUser("Dominik");
+
+//        when(mocked.getList(userService.getUserId("Dominik")));
+
+        // when
+        tut.addMovieToList("Parasite", userService.getUserId("Dominik"));
+
+        // then
+        assertThat(tut.getList(userService.getUserId("Dominik"))).hasSize(1);
+    }
+
+    @Test
+    void should_ignore_duplicated_movies() {
```

Comment:
może zamiast ignorować, to lepiej rzucić wyjątek jeśli film jest już na liście użytkownika. Do tego przydałby się identyfikator filmu. Możemy również podejść do tego tak, że dopuszczamy takie zachowanie. No, bo dlaczego nie chcemy mieć 2 filmów o takim samym tytule? Normalnie zdarzają się filmy, które mają ten sam tytuł